### ISSUE TITLE
reword downvote spec in link_spec

### ISSUE THREAD

**Issue by mant-is** [m-antis/message_bored#1]
Change

```ruby
RSpec.describe Link, '#downvote' do
  it 'decrements upvotes' do
    link = build(:link, downvotes: 1)
    link.downvote

    expect(link.downvotes).to eq 2
  end
end
```
to
```ruby
RSpec.describe Link, '#downvote' do
  it 'increments downvotes' do
    link = build(:link, downvotes: 1)
    link.downvote

    expect(link.downvotes).to eq 2
  end
end
```

<sub>be more aware when copy pasta</sub>

**Comment by m-antis** [comment]
closed by #5f9367f02d762d3a7b959181cc64f9217433ef56

**Comment by mant-is** [comment]
closed by commit [5f9367f02d762d3a7b959181cc64f9217433ef56](https://github.com/m-antis/message_bored/commit/5f9367f02d762d3a7b959181cc64f9217433ef56)
